components/Events/Card.jsx

```diff
@@ -0,0 +1,34 @@
+import styles from '/styles/Orgnjdge.module.css'
+import { TiSocialLinkedinCircular, TiSocialTwitterCircular, TiSocialFacebookCircular } from 'react-icons/ti'
```

Comment:
Change the icons to FaLinkedin, FaTwitter, FaInstagram from react-icons/fa to maintain consistency as we have not used circular icons anywhere else.